lib/components/RoundButton/RoundButton.tsx

```diff
@@ -13,6 +15,10 @@ export interface RoundButtonProps {
   className?: string;
   /** Icon name from Material Design */
   icon: string;
+  /** Icon size: sm = 32px, md = 40px, lg = 64px  */
+  size?: 'sm' | 'md' | 'lg';
```

Comment:
As the default size is now different than before this change, there will be need for UI side change to reflect this.

Reply:
RoundButton was not used in the UI, it was the RoundLinkButton version.